.circleci/config.yml

```diff
@@ -29,7 +29,7 @@ executors:
           TERM: xterm
 
 orbs:
-  compare-url: iynere/compare-url@1.0.2
+  compare-url: iynere/compare-url@1.1.0
```

Comment:
FYI: there seems to be a [different problem](https://github.com/iynere/compare-url/issues/25) with this version, it does not hurt to update though

Reply:
Progress is progress 🤷‍♂